### DISCUSSION TITLE
issues in spotify working on next js

### DISCUSSION THREAD

**Discussion by Ghufran496** [muxinc/media-chrome#848] [General]
please fix spotify audio not working on web pages
getting this error spotify-audio-element.js:117 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'createController')
    at api (spotify-audio-element.js:117:17)
    at new Promise (<anonymous>)
    at HTMLElement.load (spotify-audio-element.js:116:24)

**Comment by luwes** [comment]
duplicate of https://github.com/muxinc/media-chrome/discussions/849
